crates/pathfinder/src/core.rs

```diff
@@ -40,7 +40,8 @@ pub struct ContractCode {
 // Bytecode and entry point list of a class
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct ContractClass {
-    pub program: Vec<ByteCodeWord>,
+    // A base64 encoding of the gzip-compressed JSON representation of program.
+    pub program: String,
```

Comment:
Does this change not break `getCode -> Vec<Felt>`?

Reply:
No, because `getCode` uses the legacy `ContractCode` type as its return value (they were not the same).